core/src/main/java/org/kohsuke/stapler/TearOffSupport.java

```diff
@@ -37,12 +38,10 @@
  * @author Kohsuke Kawaguchi
  */
 public abstract class TearOffSupport {
-    private volatile Map<Class,Object> tearOffs;
+    private volatile Map<Class,Object> tearOffs = new ConcurrentHashMap<>();
```

Comment:
Chances are that this was lazy initialized to avoid allocating hash map buckets.